embedded-graphics/src/style/primitive_style.rs

```diff
@@ -79,6 +79,63 @@ where
 }
 
 /// Primitive style builder.
+///
+/// Use this builder to create styles for [primitives]. For convenience, the [`primitive_style!`]
```

Comment:
> Use this builder to create styles for [primitives].

IMO, this should link to `PrimitiveStyle` instead, which should describe how styled primitives can be used.

> For convenience, the [`primitive_style!`] macro is also provided.

It's not obvious that the function of the `primitive_style` macro is building styles. And I would move this to the end of the paragraph, because the main function of this page is describing `PrimitiveStyleBuilder`.